### PR TITLE
Fix 1.1 e2e's to work with 1.2 clusters with scheduler named "default-scheduler"

### LINKED ISSUE #19724: Change from "scheduler" to "default-scheduler" breaks backwards compatibility with e2es
**SchedulerPredicates validates that NodeSelector is respected [Conformance]**

*/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/scheduler_predicates.go:389 Expected <[]api.Event | len:0, cap:0>: [] not to be empty*

**SchedulerPredicates validates resource limits of pods that are allowed to run [Conformance]**

*/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/scheduler_predicates.go:349 Expected <[]api.Event | len:0, cap:0>: [] not to be empty*

Investigating...

### PR DESCRIPTION
**Pull request by ikehz**
Fixes #19724.

Includes a partial cherry-pick of #19608.

### CODE REVIEW

File: test/e2e/scheduler_predicates.go [hunk 1]
```diff
@@ -100,13 +109,20 @@ func verifyResult(c *client.Client, podName string, ns string, oldNotRunning int
 	expectNoError(err)
 	_, notRunningPods := getPodsNumbers(allPods)
 
+	// See comment at top of scheduler_predicates.go
```
Comment (ixdy) [inline test/e2e/scheduler_predicates.go]: repeating this three times... maybe this should be a util func?

File: test/e2e/events.go [hunk 2]
```diff
@@ -32,8 +33,18 @@ import (
 )
 
 var _ = Describe("Events", func() {
+	var c *client.Client
 	framework := NewFramework("events")
 
+	BeforeEach(func() {
```
Comment (ixdy) [inline test/e2e/events.go]: why did you add these? NewFramework already does this.

**Comment by ihmccreery** [comment]
@ixdy PTAL.

File: test/e2e/util.go [hunk 3]
```diff
@@ -879,6 +881,37 @@ func (r podResponseChecker) checkAllResponses() (done bool, err error) {
...
+	}
+	return sv.GTE(v), nil
+}
+
+
+// Multi-scheduler was implemented in #17865, first built into v1.2.0-alpha.6.
+// It changed the default scheduler name from "scheduler" to
+// "default-scheduler", breaking forward-compatibility with tests that rely on
+// finding events from the default scheduler.
+var multiSchedulerVersion = version.MustParse("v1.2.0-alpha.6")
```
Comment (ixdy) [inline test/e2e/util.go]: const?

**Comment by ixdy** [comment]
needs `gofmt`, otherwise LG

File: test/e2e/util.go (hunk 3 above)
Comment (ihmccreery) [inline test/e2e/util.go]: You can't assign a `const` a function call.

**Comment by ihmccreery** [comment]
@ixdy `gofmt` done; PTAL.

**Comment by ixdy** [comment]
LGTM
